e2e/actions.go

```diff
@@ -1,13 +1,52 @@
 package e2e
 
 import (
+	"encoding/json"
 	"log"
 	"math/rand"
 	"time"
 
 	"github.com/0xPolygon/pbft-consensus"
 )
 
+const (
+	DropNode       string = "DropNode"
+	RevertDropNode string = "RevertDropNode"
+	Partition      string = "Partition"
+	LastSequence   string = "LastSequence"
+)
+
+// MetaData is a struct that holds data about fuzz actions that happened and need to be saved in .flow file
+type MetaData struct {
+	DataType string `json:"actionType"`
+	Data     string `json:"data"`
+	Sequence uint64 `json:"sequence"`
+	Round    uint64 `json:"round"`
+}
+
+// NewMetaData creates a new MetaData object
+func NewMetaData(dataType, data string, sequence, round uint64) *MetaData {
+	return &MetaData{
+		DataType: dataType,
+		Data:     data,
+		Sequence: sequence,
+		Round:    round,
+	}
+}
+
+// ConvertActionsToByteArrays converts ActionSerializable slice to JSON representation and returns it back as slice of byte arrays
+func ConvertActionsToByteArrays(actions []*MetaData) ([][]byte, error) {
```

Comment:
Consider making this a generic functions, which would work with the slice of "objects", since it is pretty much the same as `ConvertMessagesToByteArrays` (and that one could be removed in that case). Tbh not entirely sure it would work, but it is worth of trying.

```suggestion
func ConvertToByteArrays(items []interface{}) ([][]byte, error) {
```